script/Ciel/Day 7/07_01_CIEL07_7.txt

```diff
@@ -415,7 +415,7 @@ The Acting Presbyter briefly grinds his teeth together in frustration, and then
 -- どうせロア狙いだろ？　こっちも同じだが、用があるのはヤツの工房だけだ。本体はテメェに任せてやるよ」
 // 本体 here is a bit tricky to translate, still not certain how to best phrase this.
 You're after Roa, yeah?
- Same here, but all I care about is the guy's Workshop.
+ Same here, but all I care about is the guy's workshop.
```

Comment:
Assuming this was actually intentional, Petrikow?

Reply:
Not sure if it was even me, as it appears this term missed the spreadsheet AND the glossary. I'd probably prefer it uncapitalized, honestly.